### PR TITLE
Add skyd caching env vars to docker-compose.yml

### PR DESCRIPTION
**Pull request by ChrisSchinnerl**
# PULL REQUEST

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [ ] All new methods or updated methods have clear docstrings
 - [ ] Testing added or updated for new methods
 - [ ] Verify if any changes impact the WebPortal Health Checks
 - [ ] Approriate documentation updated
 - [ ] Changelog file created

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -25,6 +25,10 @@ services:
     logging: *default-logging
     environment:
       - SIA_MODULES=gctwra
+      - SKYD_DISK_CACHE_DISABLED=true
+      - SKYD_DISK_CACHE_SIZE=53690000000
```
Comment (kwypchlo) [inline docker-compose.yml:29]:
```suggestion
      - SKYD_DISK_CACHE_SIZE=53690000000 # 50GB
```

File: docker-compose.yml [hunk 2]
```diff
@@ -25,6 +25,10 @@ services:
     logging: *default-logging
     environment:
       - SIA_MODULES=gctwra
+      - SKYD_DISK_CACHE_DISABLED=true
```
Comment (kwypchlo) [inline docker-compose.yml:28]:
```suggestion
      - SKYD_DISK_CACHE_ENABLED=false
```

Can we use enabled variable and set it to false if it's not enabled instead? The inversed env variable pattern, similar to what we do with JAEGER seems unintuitive. Also, is the disk cache enabled by default?

**Review by kwypchlo — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -25,6 +25,10 @@ services:
...
+      - SKYD_DISK_CACHE_SIZE=53690000000
+      - SKYD_DISK_CACHE_MIN_HITS=3
+      - SKYD_DISK_CACHE_HIT_PERIOD=3600
```
Comment (peterjan) [inline docker-compose.yml:31]:
```suggestion
      - SKYD_DISK_CACHE_HIT_PERIOD=3600 # 1h
```
might as well add that if you comment on the 50GiB

File: docker-compose.yml (hunk 1 above)
Comment (peterjan) [reply docker-compose.yml:29]: `nit` it's actually GiB

**Review by peterjan — DISMISSED** [review]
LGTM

**Review by peterjan — APPROVED** [review]

**Review by kwypchlo — APPROVED** [review]

**Pull request merged**